src/components/Doc/ComponentDoc.js

```diff
@@ -79,6 +87,39 @@ export const ComponentDoc = ({
         )}
       </Box>
 
+      {designSystemLink && (
+        <ThemeContext.Extend
+          value={{
+            notification: {
+              container: {
+                background: {
+                  color: 'background-contrast',
+                },
+                round: 'small',
+                pad: 'small',
+              },
+              direction: 'row',
+              message: {
+                color: 'text-strong',
+              },
+            },
+          }}
+        >
+          <Notification
+            message="Using Grommet for HPE?"
+            status="info"
+            actions={[
+              {
+                label: `See ${name} examples and guidance.`,
+                href: designSystemLink,
+                target: '_blank',
+                rel: 'noreferrer',
```

Comment:
```suggestion
```
sorry -- I changed my mind, I think noreferrer could actually be helpful from DS analytics to know how many people are coming from grommet site.